### PR TITLE
chore: add video tutorial link.

### PR DESCRIPTION
**Pull request by sayakpaul**
Reference: First part of https://github.com/huggingface/hub-docs/pull/468#pullrequestreview-1180834893

### CODE REVIEW

**Comment by merveenoyan** [comment]
@sayakpaul it's best to wait for approval before merging a PR 🙂 not a biggie though.

**Comment by sayakpaul** [comment]
@merveenoyan sorry didn't get you. I didn't merge any PRs. Could you please clarify?

File: tasks/src/video-classification/about.md
Comment (merveenoyan) [inline tasks/src/video-classification/about.md:74]:
```suggestion
```

**Review by merveenoyan — APPROVED** [review]

**Pull request merged**
